src/main/kotlin/org/opensearch/commons/alerting/AlertingPluginInterface.kt

```diff
@@ -95,6 +145,30 @@ object AlertingPluginInterface {
         )
     }
 
+    /**
+     * Get Workflow interface.
+     * @param client Node client for making transport action
+     * @param request The request object
+     * @param listener The listener for getting response
+     */
+    fun getWorkflow(
```

Comment:
we can also add execute workflow

Reply:
We could - but then we need to extract the ExecuteWorkflowRequest. 
Should I add a execute workflow method here or?